templates/shared/_kernel-support-schedule.html

```diff
@@ -1,578 +1,631 @@
-              <a href="#support-20-04" class="p-tabs__link" role="tab" id="support-20-04-tab" {% if selected_tab == "support-20-04-tab" %}aria-selected="true"{% endif %}aria-controls="support-20-04">20.04 LTS</a>
-            </li>
-            <li class="p-tabs__item" role="presentation">
-              <a href="#support-18-04" class="p-tabs__link" role="tab" id="support-18-04-tab" {% if selected_tab == "support-18-04-tab" %}aria-selected="true"{% endif %} aria-controls="support-18-04">18.04 LTS</a>
-            </li>
-            <li class="p-tabs__item" role="presentation">
-              <a href="#support-16-04" class="p-tabs__link" role="tab" id="support-16-04-tab" {% if selected_tab == "support-16-04-tab" %}aria-selected="true"{% endif %} aria-controls="support-16-04">16.04 LTS</a>
-            </li>
-            <li class="p-tabs__item" role="presentation">
-              <a href="#support-14-04" class="p-tabs__link" role="tab" id="support-14-04-tab"
-              {% if selected_tab == "support-14-04-tab" %}aria-selected="true"{% endif %}  aria-controls="support-14-04">14.04 LTS</a>
-            </li>
-          </ul>
-        </nav>
+  <div class="col-3">
+      <ul class="p-tabs--vertical on-paper js-tabbed-content u-no-margin--left u-no-padding--left" role="tablist" style="margin-top:1.5rem;">
+        <li><button class="p-tabs__item" role="tab" id="support-all-tab" aria-controls="support-all">All releases</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-lts-tab" aria-selected="false" aria-controls="support-lts">All LTS releases</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-22-04-tab" aria-selected="false" aria-controls="support-22-04">22.04 LTS</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-20-04-tab" aria-selected="false" aria-controls="support-20-04">20.04 LTS</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-18-04-tab" aria-selected="false"  aria-controls="support-18-04">18.04 LTS</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-16-04-tab" aria-selected="false"  aria-controls="support-16-04">16.04 LTS</button></li>
+        <li><button class="p-tabs__item" role="tab" id="support-14-04-tab" aria-selected="false" aria-controls="support-14-04">14.04 LTS</button></li>
+      </ul>
+    </div>
+    <div class="col-9" tabindex="0" id="support-22-04" role="tabpanel" aria-labelledby="support-22-04-tab">
+      <div class="u-hide--small" id="kernel2204"></div>
+      <table class="u-hide--medium u-hide--large">
+        <thead>
+          <tr>
+            <td>&nbsp;</td>
+            <th scope="col">Released</th>
+            <th scope="col">End of Life</th>
+            <th scope="col">Expanded security maintenance</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><strong>Ubuntu 22.04.0 LTS</strong></td>
```

Comment:
```suggestion
            <td><strong>Ubuntu 22.04.0 LTS (v5.15)</strong></td>
```
I believe these are all supposed to have the version number as well